src/ol_concourse/pipelines/infrastructure/keycloak/docker_packer_pulumi_pipeline.py

```diff
@@ -183,9 +182,7 @@ def build_keycloak_pipeline() -> Pipeline:
         ],
         image_code=keycloak_packer_code,
         packer_template_path="src/bilder/images/keycloak/keycloak.pkr.hcl",
-        env_vars_from_files={
-            "KEYCLOAK_VERSION": f"{keycloak_customization_repo.name}/.git/describe_ref"
-        },
+        env_vars_from_files={"KEYCLOAK_VERSION": KEYCLOAK_VERSION},
```

Comment:
We should probably keep the previous behavior because that is the value that we are using to set the tag on the image that we upload to dockerhub.

Reply:
https://github.com/mitodl/ol-infrastructure/blob/b6cf504e5badacb09483a45ea8de5dae991222d4/src/ol_concourse/pipelines/infrastructure/keycloak/docker_packer_pulumi_pipeline.py#L155-L157